turbopack/crates/turbopack-node/js/src/transforms/webpack-loaders.ts

```diff
@@ -165,6 +155,7 @@ type ResolveOptions = {
 
 // Patch process.env to track which env vars are read
 const originalEnv = process.env;
+// TODO: when should this be cleared?
```

Comment:
The comment a few lines further down explains why this is unfortunately never cleared:
```
      // We register the env var as dependency on the
      // current transform and all future transforms
      // since the env var might be cached in module scope
      // and influence them all
```

Reply:
thanks don't know how i missed that